graftm/run.py

```diff
@@ -37,149 +40,72 @@ def setattributes(self, args):
             self.sequence_pair_list = self.hk.parameter_checks(args)
             if hasattr(args, 'reference_package'):
                 self.p = Pplacer(self.args.reference_package)
+    
+    def _get_sequence_directions(self, search_result):     
```

Comment:
this method really tested my python understanding (particularly since there are no comments). Are you trying to make a dictionary that maps query IDs to alignment directions? If so isn't a simple for loop better, and allows you to more easily deal with name collisions?